orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/SpinnakerMetadataServerGroupTagGenerator.java

```diff
@@ -61,7 +60,9 @@
     tag.put("name", "spinnaker:metadata");
     tag.put("value", value);
 
-    return tag;
+    List<Map<String, Object>> result = new ArrayList<>();
+    result.add(tag);
+    return result;
```

Comment:
`Collections.unmodifiableList(tag)` or is that messing something up downstream?

Reply:
Should be fine - we do a `flatten` downstream but that ought not to matter